core/ledger/kvledger/txmgmt/statedb/statecouchdb/cache_test.go

```diff
@@ -57,6 +58,29 @@ func TestGetPutState(t *testing.T) {
 	require.True(t, proto.Equal(expectedValue1, v))
 }
 
+func TestGetPutStateWithBigPayloadIfKeyDoesNotExist(t *testing.T) {
+	cache := newCache(32, sysNamespaces)
+
+	// test GetState with empty cache
+	v, err := cache.getState("ch1", "ns1", "k1")
+	require.NoError(t, err)
+	require.Nil(t, v)
+
+	expectedValue := &CacheValue{Value: []byte("value")}
+	require.NoError(t, cache.putState("ch1", "ns1", "k1", expectedValue))
+
```

Comment:
After this, you could have added following to verify the initial condition for the test

```
v, err = cache.getState("ch1", "ns1", "k1")
require.NoError(t, err)
require.True(t, proto.Equal(expectedValue, v))
```